File: app/controllers/request_controller.rb
class RequestController < ApplicationController

    def new
        @member_request = MemberRequest.new
    end

    def create
        @member_request = MemberRequest.create(request_params)
        @member_request.approved = false
        if @member_request.valid? && @member_request.save
            redirect_to confirm_request_path
        end
    end

    private
    def request_params
        params.require(:member_request).permit(:first_name, :last_name, :email, :phone, :year_graduated, :address, :country, :password_pref)
    end

end

File: controllers/request_controller.js
const Request = require('../models/Request')

const getRequests = (req, res) => {
    Request.find()
        .then(response => {
            console.log(response)
            res.send(response)
        })
        .catch(err => {
            console.log(err)
            res.send(err)
        })
}

const createRequest = (req, res) => {
    const { firstName, lastName, email, phone, yearGraduated, country, address, passwordPref } = req.body

    let newRequest = new Request({ firstName, lastName, email, phone, yearGraduated, country, address, passwordPref })
    newRequest.save()
        .then(response =>{
            console.log(response)
            res.send(response)
        })
        .catch(err => {
            console.log(err)
            res.send(err)
        })
}

const deleteRequest = (req, res) => {
    Request.findByIdAndDelete(req.params.id)
        .then(response => {
            console.log(response)
            res.send(response)
        })
        .catch(err => {
            console.log(err)
            res.send(err)
        })
}

module.exports = { createRequest, getRequests, deleteRequest }